src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/producttypes/ProductTypeSyncIT.java

```diff
@@ -0,0 +1,141 @@
+import io.sphere.sdk.producttypes.queries.ProductTypeQuery;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.commercetools.sync.commons.asserts.statistics.AssertionsForStatistics.assertThat;
+import static com.commercetools.sync.integration.commons.utils.ProductTypeITUtils.deleteProductTypesFromTargetAndSource;
+import static com.commercetools.sync.integration.commons.utils.ProductTypeITUtils.populateSourceProject;
+import static com.commercetools.sync.integration.commons.utils.ProductTypeITUtils.populateTargetProject;
+import static com.commercetools.sync.integration.commons.utils.SphereClientUtils.CTP_SOURCE_CLIENT;
+import static com.commercetools.sync.integration.commons.utils.SphereClientUtils.CTP_TARGET_CLIENT;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ProductTypeSyncIT {
+
+    /**
+     * Deletes product types from source and target CTP projects.
+     * Populates source and target CTP projects with test data.
+     */
+    @Before
+    public void setup() {
+        deleteProductTypesFromTargetAndSource();
+        populateSourceProject();
+        populateTargetProject();
+    }
+
+    /**
+     * Deletes all the test data from the {@code CTP_SOURCE_CLIENT} and the {@code CTP_SOURCE_CLIENT} projects that
+     * were set up in this test class.
+     */
+    @AfterClass
+    public static void tearDown() {
+        deleteProductTypesFromTargetAndSource();
+    }
+
+    @Test
+    public void sync_WithoutUpdates_ShouldReturnProperStatistics() {
+        // preparation
+        final List<ProductType> productTypes = CTP_SOURCE_CLIENT
+            .execute(ProductTypeQuery.of())
+            .toCompletableFuture().join().getResults();
+
+        final List<ProductTypeDraft> productTypeDrafts = productTypes
+            .stream()
+            .map(ProductTypeDraftBuilder::of)
+            .map(ProductTypeDraftBuilder::build)
+            .collect(Collectors.toList());
+
+        final List<String> errorMessages = new ArrayList<>();
+        final List<Throwable> exceptions = new ArrayList<>();
+
+        final ProductTypeSyncOptions productTypeSyncOptions = ProductTypeSyncOptionsBuilder
+            .of(CTP_TARGET_CLIENT)
+            .errorCallback((error, throwable) -> {
+                errorMessages.add(error);
+                exceptions.add(throwable);
+            })
+            .build();
+
+        final ProductTypeSync productTypeSync = new ProductTypeSync(productTypeSyncOptions);
+
+        // test
+        final ProductTypeSyncStatistics productTypeSyncStatistics = productTypeSync
+            .sync(productTypeDrafts)
+            .toCompletableFuture().join();
+
+        // assertion
+        assertThat(errorMessages).isEmpty();
+        assertThat(exceptions).isEmpty();
+        assertThat(productTypeSyncStatistics).hasValues(2, 1, 0, 0);
+    }
```

Comment:
It would be good to make it consistent like other tests.

```suggestion
        assertThat(productTypeSyncStatistics
            .getReportMessage())
            .isEqualTo("Summary: 2 product types were processed in total"
                + " (1 created, 0 updated and 0 failed to sync).");
    }
```

Reply:
52bbf00